src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -107,88 +109,60 @@ const ScenarioParameters = ({
   );
 
   // Store the reset values for the run template parameters, based on defaultParametersValues and scenario data.
```

Comment:
This comment should probably be updated or removed, `generateParametersValuesFromOriginalValues` does not "store" the reset values :slightly_smiling_face: